concourse/pipelines/linux-image-build.jsonnet

```diff
@@ -44,6 +44,42 @@ local RHUIImgBuildTask(workflow, gcs_url) = daisy.daisyimagetask {
   workflow: workflow,
 };
 
+local publishresulttask = {
+  local task = self,
+
+  project:: 'gcp-guest',
+  zone:: 'us-west1-a',
```

Comment:
This may actually slow things down because the builds and therefore GCS buckets are going to be in us-central1.

Reply:
this was just from what it was already set to in the non-template version of the task. will update tho